src/README.md

```diff
@@ -5,4 +5,17 @@ algorithms.
 
 This book is a work in progress.
 
+## What do I need?
+
+The below is a quick overview of the RustCrypto organization's projects based on the use case:
+
+| Action                                              | Relevant Section                                            | Note                                                                                                                                                                 |
+| --------------------------------------------------- | ----------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Verify that data is correct                         | [Data Hashing](hashing-data/index.html)                     | Hash algorithms like SHA-256 provide a one-way map of data to a fixed-length that can be easily compared                                                             |
+| Obscure a password                                  | [Password Hashing](hashing-password/index.html)             | Data hash algorithms are designed to be fast; password hash algorithms like `bcrypt`are designed to be slow. Use these to store password results in a database.      |
```

Comment:
```suggestion
| Authenticate users with a password                   | [Password Hashing](hashing-password/index.html)             | Password hashing algorithms are designed to be slow to prevent brute force attacks. Password hashes can be stored in a database.      |
```